src/ui/popup/settings-page/devtools.tsx

```diff
@@ -7,14 +7,27 @@ import {ViewProps} from '../types';
 import {isFirefox} from '../../../utils/platform';
 
 function getExistingDevToolsWindow() {
-    return new Promise<chrome.windows.Window>((resolve) => {
-        chrome.windows.getAll({
-            populate: true,
-            windowTypes: ['popup']
-        }, (w) => {
-            for (const window of w) {
-                if (window.tabs[0].url.endsWith('ui/devtools/index.html')) {
-                    resolve(window);
+    if (!isMobile()) {
+        return new Promise<chrome.windows.Window>((resolve) => {
+            chrome.windows.getAll({
+                populate: true,
+                windowTypes: ['popup']
+            }, (w) => {
+                for (const window of w) {
+                    if (window.tabs[0].url.endsWith('ui/devtools/index.html')) {
+                        resolve(window);
+                        return;
+                    }
+                }
+                resolve(null);
+            });
+        });
+    }
+    return new Promise<chrome.tabs.Tab>((resolve) => {
+        chrome.tabs.query({}, (t) => {
+            for (const tab of t) {
+                if (tab.url.endsWith('ui/devtools/index.html')) {
```

Comment:
Create a function like `isDevToolsTab(tab)` and use it in all places where necessary.